resources/views/components/thread.blade.php

```diff
@@ -13,30 +13,30 @@
             :in-thread="true"
             :key="'question-'.$rootId"
         />
+
+        @if($grandParentId !== null && ($parentId === null || $grandParentId !== $rootId))
```

Comment:
here we are adding support for following (when root is from following user and parent is not from following user it adds view more comments divider in thread.)

Reply:
What `grandParentId` means?

Reply:
> What `grandParentId` means?

The question's root and the question's parent's patent (means grand parent) both are the same we do not show 'view more comments' as there are no comments between both questions.

Reply:
Or it should be based on comments count on root? 🤔